drivers/exec/driver_test.go

```diff
@@ -251,6 +254,104 @@ func TestExecDriver_StartWaitRecover(t *testing.T) {
 	require.NoError(harness.DestroyTask(task.ID, true))
 }
 
+func TestExecDriver_DestroyKillsAll(t *testing.T) {
```

Comment:
I'm a sucker for succinct descriptions on tests, but not a blocker.

```suggestion
// TestExecDriver_DestroyKillsAll asserts that when TaskDestroy is called all
// task processes are cleaned up.
func TestExecDriver_DestroyKillsAll(t *testing.T) {
```